applications/nrf_desktop/configuration/nrf52840_pca10056/pm_static_ZDebugMCUBoot.yml

```diff
@@ -0,0 +1,31 @@
+mcuboot:
+  address: 0x0
+  size: 0xc000
+mcuboot_pad:
+  address: 0xc000
+  size: 0x200
+app:
+  address: 0xc200
+  size: 0x69e00
+mcuboot_primary:
+  orig_span: &id001
+  - mcuboot_pad
+  - app
+  span: *id001
+  address: 0xc000
+  size: 0x6a000
+mcuboot_primary_app:
+  orig_span: &id002
+  - app
+  span: *id002
+  address: 0xc200
+  size: 0x69e00
+mcuboot_secondary:
+  address: 0x76000
+  size: 0x6a000
+mcuboot_scratch:
```

Comment:
Scratch is no longer used by MCUBoot (this is default for all nRF family SoC). This is changed by BOOT_SWAP_USING_MOVE option in MCUBoot Kconfig. In that case we should disable this option or no longer configure scratch size.

Reply:
This applies to all the MCUBoot configurations.

Reply:
Since we switch to B0 let's leave it as is - subsequent PR will remove mcuboot altogether.